distributed/utils_comm.py

```diff
@@ -204,9 +204,9 @@ def _namedtuple_packing(o: Any, handler: Callable[..., Any]) -> Any:
 
 
 def _unpack_remotedata_inner(
-    o: Any, byte_keys: bool, found_keys: set[WrappedKey]
+    o: Any, byte_keys: bool, found_futures: set[WrappedKey]
```

Comment:
nit: I'ts a bit confusing that `found_futures` is a set of (essentially) `Key`s.